context-free.md

```diff
@@ -2,21 +2,61 @@
 
 ##  Who is this programming language for?
 
+Context Free is for artists iterested in exploring generative art.
 
 ## What is easy to do in this language? Why is it easy?
 
+It's easy to generate art that has a lot of symmetry. One of the easiest tricks
+for generating an image is to create recursive shape rules, and this tends to
+lead to shapes with a recursive structure, which results in symmetry. The
+handful of tree-like example programs confirm this.
 
 ## What is hard to do in this language? Why is it hard?
 
+It would be hard to replicate a picture of some real object in the language.
+Encoding the information in a picture (which probably does not have too much
+symmetry) would take a lot of code in this language, whose primary strength is
+encoding symmetry in only a few lines.
 
-## How did you learn how to program in this language?
+Obviously, it would also be tremendously difficult to do any sort of
+computation or general-purpose programming in this language (although not
+impossible; the language has conditionals, loops, variables, functions, and
+recursion, so I believe it's Turing-complete).
```

Comment:
As you'll learn in this class, Turing-completeness isn't everything (and also trying to actually program anything in a language just because it is Turing-complete is never fun). As an example, https://github.com/jbangert/trapcc and https://github.com/xoreaxeaxeax/movfuscator are super cool projects I would never want to develop.